experimental/overhead_matching/swag/scripts/lambda_cloud/lambda_api/client.py

```diff
@@ -0,0 +1,356 @@
+"""Lambda Cloud API client."""
+
+import os
+import time
+import logging
+import base64
+from typing import List, Dict, Optional, Any
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+try:
+    from .models import (
+        Instance, InstanceType, InstanceStatus, SSHInfo,
+        LambdaCloudError, InsufficientCapacityError, RateLimitError
+    )
+except ImportError:
+    from models import (
+        Instance, InstanceType, InstanceStatus, SSHInfo,
+        LambdaCloudError, InsufficientCapacityError, RateLimitError
+    )
```

Comment:
`from experimental.overhead_matching.swag.scripts.lambda_cloud.lambda_api.models import ...`

Reply:
also consider moving this to `common/lambda`